### PR TITLE
fix: correct auto land light

### LINKED ISSUE #5090: AUTO LAND light is backwards
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-06-10T09:49:19+00:00",
  "ref": "master",
  "sha": "1578f4fa852800ef1d7c46ad6460f21bf92a2c3a",
  "actor": "MisterChocker",
  "event_name": "manual"
}
```

### Describe the bug

Autoland light is backwards

### Expected behavior

Texture should be red background with black words

### Steps to reproduce

1. Turn on the light test and compare with the photo

### References (optional)

image

### Additional info (optional)

_No response_

### Discord Username (optional)

Illuminati

### PR DESCRIPTION
**Pull request by tyler58546**
Fixes #5090

## Summary of Changes

- Change autoland warning light texture to match IRL as closely as possible, and flash
- Fix amber x-feed open label (reverted to old green texture)
  - Add new amber OPEN label for cockpit door

## Screenshots (if necessary)

All AC bus happy:

https://user-images.githubusercontent.com/9995998/123266965-9a6f0880-d550-11eb-81f7-2e58731838d9.mp4

AC bus 2 dead (I also checked with AC ESS SHED dead but didn't record):

https://user-images.githubusercontent.com/9995998/123266555-406e4300-d550-11eb-8189-2abc73b3a77b.mp4

image
image

## Additional context

Discord username (if different from GitHub):

## Testing instructions

See https://github.com/flybywiresim/a32nx/pull/5100#issuecomment-860660702 for power sources of lights.

To test with AC bus 2 off:
 - With engines running and no APU, turn off bus tie, and GEN 2
To test with AC ESS SHED off:
- With engines running and no APU, turn off bus tie, and GEN 1... you have 3 seconds to observe the lights before AC ESS SHED switches over

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by Illuminati270** [comment]
Well done.

Checked, Auto Land light working correctly! Other light textures also checked to their true colors.

File: src/behavior/src/A32NX_Interior_FCU.xml [hunk 1]
```diff
@@ -317,15 +317,23 @@
...
-            <NO_SEQ2 />
+            <NO_SEQ1/>
         </DefaultTemplateParameters>
 
         <UseTemplate Name="FBW_Push_Held">
-            <SEQ1_CODE>(L:A32NX_AUTOPILOT_AUTOLAND_WARNING, bool)</SEQ1_CODE>
-            <SEQ1_EMISSIVE_DRIVES_VISIBILITY>True</SEQ1_EMISSIVE_DRIVES_VISIBILITY>
+            <NO_SEQ1/>
+            <SEQ2_CODE>(L:A32NX_AUTOPILOT_AUTOLAND_WARNING, bool) (E:ABSOLUTE TIME, seconds) 1 % 0.4 &gt; and</SEQ2_CODE>
+            <SEQ2_EMISSIVE_DRIVES_VISIBILITY>True</SEQ2_EMISSIVE_DRIVES_VISIBILITY>
```
Comment (davidwalschots) [inline src/behavior/src/A32NX_Interior_FCU.xml:326]: Usage of this template is as follows:
```xml
            <UseTemplate Name="FBW_Airbus_Autoland_Warning">
                <SIDE>L</SIDE>
                <!-- Note there is only a single emissive. Once two emissives are available SEQ1 should use AC ESS SHED, and SEQ 2 should use AC 2. -->
                <SEQ_POWERED>(L:A32NX_ELEC_AC_ESS_SHED_BUS_IS_POWERED, Bool) (L:A32NX_ELEC_AC_2_BUS_IS_POWERED, Bool) or</SEQ_POWERED>
            </UseTemplate>
            <UseTemplate Name="FBW_Airbus_Autoland_Warning">
                <SIDE>R</SIDE>
                <!-- Note there is only a single emissive. Once two emissives are available SEQ1 should use AC ESS SHED, and SEQ 2 should use AC 2. -->
                <SEQ_POWERED>(L:A32NX_ELEC_AC_ESS_SHED_BUS_IS_POWERED, Bool) (L:A32NX_ELEC_AC_2_BUS_IS_POWERED, Bool) or</SEQ_POWERED>
            </UseTemplate>
```
Previously @komp1821 indicated there were two lights. Now it seems there's only one light. I think it's best to ask @komp1821 which bus(es) actually power that light, to ensure correctness.

**Review by davidwalschots — CHANGES_REQUESTED** [review]

**Comment by marcman86** [comment]
Quality Assurance Tester Report
Discord:marcman86#4907
Object of testing: #5100
Tier of Testing: 1
Date: 13/06/2021

Testing Process: 
- Spawn C&D at EDDP
- Turn on bat. and ext. power
- ANN LT TEST

Negatives: -
Testing Results: Passed
Conclusions: Works
Media:





Hope this helps :-)

**Review by aguther — APPROVED** [review]
LGTM beside the comments from @davidwalschots. The test function can be added in a separate PR.

**Comment by davidwalschots** [comment]
@marcman86 what about the flashing? 😃

**Comment by marcman86** [comment]
OK. Tryed it, but dont get it to flash. 

- C&D spawn
- Turn on bat. and ext. power
- Align ADIRS
- Press and hold AUTOLAND light button (tryed both sides)

**Comment by aguther** [comment]
> OK. Tryed it, but dont get it to flash.
> 
> * C&D spawn
> * Turn on bat. and ext. power
> * Align ADIRS
> * Press and hold AUTOLAND light button (tryed both sides)

Up to now the test function is not yet implemented (see my last comment). This PR is focused on the look of the AUTOLAND warning light + flashing (it was steady before).

The warning is armed when:
- below 200 ft RA
- at least one AP engaged
- FMA shows LAND or FLARE

The warning when armed is triggered when:
- both AP are disengaged
- above 15 ft RA LOC is lost or deviation is greater than 1/4 dot (0.2°)
- above 100 ft RA G/S is lost or deviation is greater than 1 dot (0.4°)

**Comment by marcman86** [comment]
> 
> 
> > OK. Tryed it, but dont get it to flash.
> > 
> > * C&D spawn
> > * Turn on bat. and ext. power
> > * Align ADIRS
> > * Press and hold AUTOLAND light button (tryed both sides)
> 
> Up to now the test function is not yet implemented (see my last comment). This PR is focused on the look of the AUTOLAND warning light + flashing (it was steady before).
> 
> The warning is armed when:
> 
>     * below 200 ft RA
> 
>     * at least one AP engaged
> 
>     * FMA shows LAND or FLARE
> 
> 
> The warning when armed is triggered when:
> 
>     * both AP are disengaged
> 
>     * above 15 ft RA LOC is lost or deviation is greater than 1/4 dot (0.2°)
> 
>     * above 100 ft RA G/S is lost or deviation is greater than 1 dot (0.4°)

Tested this. It did flash. Nice work guys

File: src/behavior/src/A32NX_Interior_FCU.xml (hunk 1 above)
Comment (bouveng) [reply src/behavior/src/A32NX_Interior_FCU.xml:326]: Hi! Seems like they are powered by both? Makes sense from redundant perspective. 

bus

**Comment by TheSquirtingPantin** [comment]
Quality Assurance Tester

Discord: Remi#8871
Object of testing: #5100
Tier of Testing: 1
Date: 13/06/2021

Testing Process:
1) Full Flight KMSY-KMIA
2) ANN LT Test + visual assessment of modified lights.
3) Visual assessment of flashing during AUTOLAND capability lost

Negatives: None.
Testing Results: Passed. 

Conclusions: LGTM. 
Media: N/A

**Comment by komp1821** [comment]
There you go!

Screenshot 2021-06-14 145327

**Comment by tracernz** [comment]
FYI: this is held up to devise a technique to separately power the two halves while still looking decent. Should have time this week.

File: src/behavior/src/A32NX_Interior_Generics.xml [hunk 2]
```diff
@@ -322,14 +322,22 @@
             - HOLD_SIMVAR                       <no default>    SimVar to contain the held state
             - SEQ1_CODE                         Not present     Sequence 1 emissive code. If not present, the SEQ1 emissive will also not respond to the ANN LT TEST
             - SEQ2_CODE                         Not present     Sequence 2 emissive code. If not present, the SEQ2 emissive will also not respond to the ANN LT TEST
+            - SEQ3_CODE                         Not present     Sequence 3 emissive code. If not present, the SEQ2 emissive will also not respond to the ANN LT TEST
+            - SEQ4_CODE                         Not present     Sequence 4 emissive code. If not present, the SEQ2 emissive will also not respond to the ANN LT TEST
```
Comment (davidwalschots) [inline src/behavior/src/A32NX_Interior_Generics.xml:326]:
```suggestion
            - SEQ3_CODE                         Not present     Sequence 3 emissive code. If not present, the SEQ3 emissive will also not respond to the ANN LT TEST
            - SEQ4_CODE                         Not present     Sequence 4 emissive code. If not present, the SEQ4 emissive will also not respond to the ANN LT TEST
```

**Review by davidwalschots — APPROVED** [review]

**Comment by davidwalschots** [comment]
Interesting emissive code. Looked cool in the videos you sent me!

**Comment by sidnov** [comment]
QA Report

Object of testing: #5100
Tier of Testing: 1
Date: 25/06/2021

Testing Process:
Spawned on runway, check autoland light with ANN LT TEST. Observed XFEED OPEN is green, and Cockpit Door OPEN is amber. Tested with AC BUS 2 and ESS SHED off separately and the correct halves within the button shutdown to show accurate illumination. 

Negatives: None
Testing Results: Passed

Conclusions: Works well overall. LGTM!

**Pull request merged**